src/motd.cpp

```diff
@@ -124,9 +128,9 @@ extern "C" char *fetchMOTD(int cache)
 
 #else
 
-extern "C" char *fetchMOTD(int /* cache */)
+extern "C" char *fetchMOTD(int /* cache */, int enable_motd)
```

Comment:
This will trigger an unused parameter warning, comment like with the cache parameter